docker/docker-compose.yml

```diff
@@ -22,7 +34,24 @@ services:
       timeout: 5s
       retries: 5
   postgres:
-    image: postgres:15
+    image: docker.io/library/postgres:15
+    cap_drop:
+      - ALL
+    cap_add:
+      - CHOWN
+      - DAC_READ_SEARCH
+      - FOWNER
+      - SETGID
+      - SETUID
+    security_opt:
+      - no-new-privileges:true
+    # uncomment to set resource usage limits
```

Comment:
Can you remove the resource usage limits? Anyone who needs it can configure it themselves.